apps/explorer/src/components/ownedobjects/views/ObjectFieldsCard.tsx

```diff
@@ -0,0 +1,249 @@
+                                                            'mt-0.5 block w-full cursor-pointer rounded-md border px-1.5 py-2 text-left text-body',
+                                                            active
+                                                                ? 'border-transparent bg-gray-40 text-steel-darker'
+                                                                : 'border-transparent bg-white font-medium text-steel-darker'
+                                                        )}
+                                                    >
+                                                        {name}
+                                                    </button>
+                                                )}
+                                            </Combobox.Option>
+                                        ))}
+                                    </Combobox.Options>
+                                </Combobox>
+                                <div className="max-h-[600px] min-h-full overflow-auto overflow-x-scroll py-3">
+                                    <VerticalList>
+                                        {fieldsNames.map(([name, value]) => (
+                                            <div
+                                                key={name}
+                                                className="mx-0.5 mt-0.5 md:min-w-fit"
+                                            >
+                                                <ListItem
+                                                    active={
+                                                        activeFieldName === name
+                                                    }
+                                                    onClick={() =>
+                                                        onChangeField(name)
+                                                    }
+                                                >
+                                                    <div className="flex justify-between">
+                                                        <Text
+                                                            variant="body/medium"
+                                                            color="steel-darker"
+                                                        >
+                                                            {name}
+                                                        </Text>
+                                                        <div className="capitalize">
+                                                            <Text
+                                                                variant="subtitle/normal"
+                                                                color="steel"
+                                                            >
+                                                                {typeof value}
```

Comment:
This is fine for now, but what we actually want here is to use `getNormalizedMoveStruct` to get the actual move types for this.